packages/web-react/src/components/Toast/README.md

```diff
@@ -102,25 +102,41 @@ sorted from top to bottom for the `top` vertical alignment, and from bottom to t
 👉 Please note the _actual_ order in the DOM is followed when users tab over the interface, no matter the _visual_
 order of the toast queue.
 
-#### Toast Queue Limitations
+#### Collapsing
+
+The collapsible Toast queue is turned on by default and can hold up to 3 ToastBar components.
+When the queue is full, the oldest ToastBar components are collapsed at the start of
+the queue and are only accessible by closing the newer ones.
```

Comment:
`newer` -> maybe `latest`?

Reply:
I feel it like _latest_ is a subset of _newer_. And you need to close all newer notifications, not just the latest 🙂 🎉.

Reply:
Ok.